### PR TITLE
Exclude SLE12 platform.

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM
You forgot to mention that it depends on https://github.com/crowbar/barclamp-crowbar/pull/1175

**Comment by jsuchome** [comment]
+1

File: crowbar_framework/app/models/ceilometer_service.rb [hunk 1]
```diff
@@ -32,21 +32,33 @@ def role_constraints
       {
         "ceilometer-agent" => {
           "unique" => false,
-          "count" => -1
+          "count" => -1,
+          "exclude_platform" => {
+            "suse" => "12.0"
```
Comment (vuntz) [inline crowbar_framework/app/models/ceilometer_service.rb]: This is not correct: ceilometer-agent should actually be used on SLE12 compute nodes.

Comment (jsuchome) [inline crowbar_framework/app/models/ceilometer_service.rb]: Both SLE11 and SLE12

Comment (aplanas) [inline crowbar_framework/app/models/ceilometer_service.rb]: I will fix it. In the task description this was not noted, but can be deduced becasue Nova can go there.

**Comment by aplanas** [comment]
* Remove constraint from ceilometer-agent

**Pull request merged**